resources/Makefile.erb

```diff
@@ -1,11 +1,15 @@
 <% dirnames = get_directories.map {|d| d.path } %>
-export SHELL := $(shell which bash)
+
+SHELL = $(PWD)/profiling_shell.sh
+export VANAGON_PROJECT ?= <%= @name %>
+export VANAGON_PLATFORM ?= <%= @platform.name.tr('-','.') %>
 
 tempdir := $(shell mktemp -d -p /var/tmp 2>/dev/null || mktemp -d -t 'tmp')
 workdir := $(PWD)
 
 all: file-list-before-build <%= package_name %>
 
+<%= package_name %>: export VANAGON_TARGET = create-package
```

Comment:
I think I misspoke about the unpack. I believe the unpack happens in the makefile now that I think about it, but downloading the sources and installing build time dependencies happens outside of the makefile